src/java/org/apache/cassandra/cache/ChunkCache.java

```diff
@@ -172,7 +171,7 @@ public void onRemoval(Key key, Buffer buffer, RemovalCause cause)
         buffer.release();
     }
 
-    public void close()
+    public void clear()
```

Comment:
This change is not related to the issue, right?

Reply:
I cant find any usage of that method. You are technically right, yes. But looking at it, why not to rename it to clear? The impl invalidates the cache which is kind of a clearing ... I think "clear" makes more sense and reflects what that method does than "close". That cache does not implement (Auto)Closeable or anything like that, there is nothing to _close_.